core/arch/arm/plat-k3/drivers/sec_proxy.h

```diff
@@ -1,47 +0,0 @@
-/* SPDX-License-Identifier: BSD-2-Clause */
-/*
- * Texas Instruments K3 Secure Proxy Driver
- *
- * Copyright (C) 2022 Texas Instruments Incorporated - https://www.ti.com/
- *	Manorit Chawdhry <m-chawdhry@ti.com>
- */
-
-#ifndef SEC_PROXY_H
-#define SEC_PROXY_H
-
-#include <stddef.h>
-#include <stdint.h>
-#include <tee_api_types.h>
-
-#define SEC_PROXY_MAX_MSG_SIZE 60
```

Comment:
Technically this refactor is changing a behaviour so shouldn't be done in a single commit, this is a refactor and a bug fix where you are modifying the sec proxy message size and starting to use another macro with another value itself. Put all the bug fixes in the beginning of the PR and then do refactors. The refactor commits should just refactor and not change the behaviour of the code.